pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -116,21 +116,35 @@ public QueryExecutor openConnectionImpl(HostSpec[] hostSpecs, String user, Strin
     int connectTimeout = PGProperty.CONNECT_TIMEOUT.getInt(info) * 1000;
 
     // - the targetServerType setting
-    HostRequirement targetServerType;
+    HostRequirement requestedTargetServerType;
```

Comment:
I think, one can extract the getting of 'requestedTargetServerType' (lines 119-127) into private method

Reply:
@AlexElin  Had changed